telegram_phone_number_checker/main.py

```diff
@@ -80,6 +90,32 @@ async def get_names(client: TelegramClient, phone_number: str) -> dict:
                     "phone": user.phone,
                 }
             )
+        if download_profile_photos is True:
+            try:
+                photo_output_path = Path("{}_{}_photo.jpeg".format(user.id, user.phone))
```

Comment:
It looks like `user.phone` is often `None`, let's use the user supplied `phone_number` instead